### PR TITLE
dumpzilla: add page

### PR DESCRIPTION
**Pull request by ciph3rz**
- [x ] The page (if new), does not already exist in the repository.
- [ x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x ] The page has 8 or fewer examples.
- [ x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x ] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x ] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

**Comment by ciph3rz** [comment]
#6863

File: pages/common/dumpzilla.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+- Extract everything except the DOM data:
+
+`dumpzilla {{path/to/browser_profile}} --All`
+
+- Extract all stored passwords in browser profile:
+
+`dumpzilla {{path/to/browser_profile}} --Passwords`
+
+- Extract complete listing of browser download history:
```
Comment (navarroaxel) [inline pages/common/dumpzilla.md:18]: Where is stored?

File: pages/common/dumpzilla.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# dumpzilla
+
+> Forensic tool to extract information from Firefox, Iceweasel and Seamonkey browsers.
+> More information: <https://www.dumpzilla.org>.
+
+- Gather summary data from browser profile:
```
Comment (navarroaxel) [inline pages/common/dumpzilla.md:6]: Print a summary or it's dumped to a file?

File: pages/common/dumpzilla.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+- Extract all stored passwords in browser profile:
+
+`dumpzilla {{path/to/browser_profile}} --Passwords`
+
+- Extract complete listing of browser download history:
+
+`dumpzilla {{path/to/browser_profile}} --Downloads`
+
+- Extract complete listing of browser bookmarks:
```
Comment (einverne) [inline pages/common/dumpzilla.md:22]: Same here,  store in the current directory?

**Review by einverne — CHANGES_REQUESTED** [review]

**Pull request closed without merging**